src/mavesp8266_parameters.cpp

```diff
@@ -41,7 +41,7 @@
 #include "mavesp8266_parameters.h"
 #include "crc.h"
 
-const char* kDEFAULT_SSDI       = "PixRacer";
+const char* kDEFAULT_SSID       = "PixRacer";
```

Comment:
I'm so dyslexic I had to look at this 3 or 4 times before I could see the difference :)